databases/week2/part3_morequeries.sql

```diff
@@ -0,0 +1,42 @@
+-- Part 3: More queries
+
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT * FROM task
+WHERE title LIKE '%SQL%' OR description LIKE '%SQL%';
+
+SELECT 
+   task.title, status.name AS status 
+FROM task
+JOIN status
+ON status.id = task.status_id
+
+SELECT * FROM task
+JOIN user 
+on user.id = task.user_id
+WHERE user.email LIKE '%@spotify.com' 
+
+-- Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT * FROM task
+JOIN user 
+on user.id = task.user_id
+WHERE user.name = 'Donald Duck' AND task.status_id = 1;
```

Comment:
dont hardcode the status_id. Here do a join and look for the status 'Not started'